docs/_source/docs/stack_group_config.rst

```diff
@@ -74,6 +75,19 @@ Extension can be ``json`` or ``yaml``.
 Note that if ``template_bucket_name`` is not supplied, this parameter is
 ignored.
 
+j2_environment
+~~~~~~~~~~~~~~~~~~~
+
+A dictionary that is combined with the default jinja2 environment.
+It's converted to keyword arguments then passed to [jinja2.Environment](https://jinja.palletsprojects.com/en/2.11.x/api/#jinja2.Environment).
+
+```yaml
+j2_environment:
+  extensions:
+    - jinja2.ext.i18n
+    - jinja2.ext.do
```

Comment:
would you mind giving an example of passing in a few other [jinja2 environment options](https://jinja.palletsprojects.com/en/3.0.x/api/#jinja2.Environment) ?